.github/workflows/build-41.yml

```diff
@@ -1,15 +1,44 @@
 name: ublue main 41
+
 on:
   pull_request:
   merge_group:
   schedule:
     - cron: '5 4 * * *'  # 4am-ish UTC everyday (timed against official fedora container pushes)
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref || github.run_id }}
+  cancel-in-progress: true
+
 jobs:
   build:
     name: build
     uses: ./.github/workflows/reusable-build.yml
     secrets: inherit
+    strategy:
+      fail-fast: false
+      matrix:
+        image_name:
+          - base
+          - silverblue
+          - kinoite
+          - sericea
+          - onyx
+          - lazurite
+          - vauxite
+        include:
+          - image_name: base
+            source_image_name: base-atomic
+          - image_name: onyx
+            source_image_name: budgie-atomic
+          - image_name: sericea
+            source_image_name: sway-atomic
+          - image_name: lazurite
+            source_image_name: lxqt-atomic
+          - image_name: vauxite
+            source_image_name: xfce-atomic
```

Comment:
From F42+, we can switch to the *-atomic naming wherever used upstream, so this will only be used in F41 workflows.